.github/.github.csproj

```diff
@@ -1,5 +0,0 @@
-﻿<Project Sdk="Microsoft.Build.NoTargets/1.0.94">
-  <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-  </PropertyGroup>
-</Project>
```

Comment:
This file may not matter to you in VS Code, but over here in full IDE land we do use it =P